topics/dot-net/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: dot-net-framework, microsoft-dot-net
+created_by: Microsoft
+display_name: .NET
+logo: dot-net.png
+released: February 13 2002
+short_description: .NET framework is developed by Microsoft which helps developers to create applications more easily.
+topic: dot-net
+related: software, framework
```

Comment:
I'd remove software here. A lot of things can be software. A programming language can be software. Git can be software. Laravel can be software. In fact, all of the topics that can be classified as "software" aren't tagged software, so I don't think `dot-net` should be an exception. We are on GitHub after all, the place "where software is built".